shotover-proxy/src/transforms/mod.rs

```diff
@@ -217,6 +222,7 @@ pub enum TransformsConfig {
     CassandraSinkSingle(CassandraSinkSingleConfig),
     RedisSinkSingle(RedisSinkSingleConfig),
     KafkaSink(KafkaSinkConfig),
+    CassandraPeersRewrite(CassandraPeersRewriteConfig),
```

Comment:
We either need to add `#[cfg(feature = "alpha-transforms")]` or document the functionality in transforms.md

Reply:
I've got a docs and example ready in this branch: https://github.com/conorbros/shotover-proxy/tree/cassandra-docs